airgun/entities/host.py

```diff
@@ -22,8 +23,15 @@ def create(self, values):
         view = self.navigate_to(self, 'New')
         view.fill(values)
         view.submit.click()
-        view.flash.assert_no_error()
-        view.flash.dismiss()
+        host_view = HostDetailsView(self.browser)
+        wait_for(
+            lambda: host_view.is_displayed is True,
+            timeout=300,
+            delay=10,
```

Comment:
Does host creation take that long? I'd probably reduce to 1 min with delay of 1-2 sec, waiting for 10 seconds just for creation seems too much for me.